src/Command/Self/UpdateCommand.php

```diff
@@ -124,6 +124,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // can't read files properly from the overwritten Phar, or perhaps it's
         // because the autoloader's name has changed. We avoid the problem by
         // terminating now.
-        exit;
+        throw new Exception('Unexpected instantiation');
```

Comment:
If we do not have an exit here the self-update command output an exception after updating the project.